File: services/main/frontend/lib/src/pages/project-build-jobs/project-build-job-status-badge/project_build_job_status_badge_component.dart
import 'package:angular/angular.dart';
import 'dart:html';
import '../../../model/core.dart';

@Component(
    selector: 'project-build-job-status-badge',
    templateUrl: 'project_build_job_status_badge_component.html',
    directives: const [
      coreDirectives,
    ]
)
class ProjectBuildJobStatusBadgeComponent {

  @Input()
  PyroWorkspaceImageBuildJob job;
}

File: services/main/frontend/src/app/features/projects/pages/build-jobs/components/project-build-job-status-badge/project-build-job-status-badge.component.ts
import { Component, Input } from '@angular/core';
import { WorkspaceImageBuildJob } from '../../../../../../core/models/workspace-image-build-job';

@Component({
  selector: 'cc-project-build-job-status-badge',
  templateUrl: './project-build-job-status-badge.component.html',
  styleUrls: ['./project-build-job-status-badge.component.css']
})
export class ProjectBuildJobStatusBadgeComponent {

  @Input()
  job: WorkspaceImageBuildJob;
}
